File: api/src/main/java/com/ungalthedal/api/model/Shared.java
package com.ungalthedal.api.model;

import com.ungalthedal.api.domain.admin.category.CategoryListView;
import com.ungalthedal.api.domain.admin.category.category.CategoryView;
import com.ungalthedal.api.domain.admin.category.sub_category.SubCategoryView;
import com.ungalthedal.api.domain.admin.location.LocationListView;
import com.ungalthedal.api.domain.admin.location.city.CityView;
import com.ungalthedal.api.domain.admin.location.country.CountryView;
import com.ungalthedal.api.domain.admin.location.district.DistrictView;
import com.ungalthedal.api.domain.admin.location.region.RegionView;
import com.ungalthedal.api.domain.admin.location.state.StateView;
import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Shared {
    private Long contactCountByReadInd;
}

File: ui/src/app/core/models/shared/Shared.ts
import { Category } from "../admin/category/Category";
import { CategoryList } from "../admin/category/CategoryList";
import { SubCategory } from "../admin/category/SubCategory";
import { City } from "../admin/location/City";
import { Country } from "../admin/location/Country";
import { District } from "../admin/location/District";
import { LocationList } from "../admin/location/LocationList";
import { Region } from "../admin/location/Region";
import { State } from "../admin/location/State";

export class Shared {
    cityList: City[] = [];
    districtList: District[] = [];
    stateList: State[] = [];
    countryList: Country[] = [];
    regionList: Region[] = [];
    categoryList: Category[] = [];
    subCategoryList: SubCategory[] = [];
    locationViewList: LocationList[] = [];
    categoryViewList: CategoryList[] = [];
    contactContactByReadInd: number = 0;
}
